vdr/doc-updater.go

```diff
@@ -17,12 +17,14 @@ type DocUpdater struct {
 	KeyCreator nutsCrypto.KeyCreator
 	// VDR is used for resolving before and updating DID documents after the operation has been performed
```

Comment:
not for resolving, since that's what `DocResolver` now is for ("what's up, doc?")